src/github-cli/install.sh

```diff
@@ -9,6 +9,7 @@
 
 CLI_VERSION=${VERSION:-"latest"}
 INSTALL_DIRECTLY_FROM_GITHUB_RELEASE=${INSTALLDIRECTLYFROMGITHUBRELEASE:-"true"}
+TEST_EXPORT=${TEST_EXPORT:-}
```

Comment:
> Add ability to export bash functions for testing: TEST_EXPORT=find_version_from_git_tags ./src/github-cli/install.sh

Could you help me explain the scenarios in which this is helpful?  I read this part ☝️ , but usually tests for Features are run with `devcontainers features test` cmd. Curious to understand your thought behind it's usage 😄

Reply:
I used this "export" mechanism in my development to manually test a single bash function from the `install.sh` script without running the whole `install.sh` script (which wouldn't even work on macOS). I noticed that you copied over bash functions for testing into the `test.sh` script, and with this mechanism you could possibly export functions for testing in CI rather that copying over the functions. However, I haven't made use of that in the current PR, and I'm not really attached to the TEST_EXPORT mechanism either. We could also just drop it from the final implementation.

Reply:
Makes sense. I agree that if we aren't planning to make use of it, it's great to drop the changes. thanks!